src/Commands/Backup/ListCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * @file
+ * Contains Pantheon\Terminus\Commands\SSHKey\ListCommand
+ */
+
+
+namespace Pantheon\Terminus\Commands\Backup;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Terminus\Collections\Sites;
+use Terminus\Models\Environment;
+
+class ListCommand extends TerminusCommand
+{
+
+    /**
+     * Contructor that supports injection for unit testing.
+     */
+    public function __construct($sites = null)
+    {
+        parent::__construct();
+        $this->sites = $sites ? $sites : new Sites();
+    }
+
+    /**
+     * Lists the Backups for a given Site and Environment
+     *
+     * @authorized
+     *
+     * @command backup:list
+     * @aliases backups
+     *
+     * @param string $environment Name of the environment to retrieve
+     * @param string $element Element filter code/files/database/db (optional)
```

Comment:
Or more like you have below even. I think I've seen optional args bracketed like string here:
`* @param [string] $type [code|files|database|db] Only show backups of a certain type`